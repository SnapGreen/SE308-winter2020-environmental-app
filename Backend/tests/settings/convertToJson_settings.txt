settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/scripts/settings.txt
	AWKDIR: awk/
	DATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/
	FDADIR: FDA/
	FDAINDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/FDA/
	FDAOUTDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/
	LOGDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/logs/
	RAWDATADIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/
	TMPDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/
	TESTDIR: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/
	FDAINFILE: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/dataraw/FDA/branded_food.csv
	MAPFILE: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/data/FDA/map_fdcid_gtin.csv
	PRODS_PER_JSON: 250
	FDA_PATTERNFILE: fda_removalpatterns.txt
	SPLIT_PREFIX: branded_food_
	SUFFIX_LEN: 4
	OUTFILE_END: .json
	TMPFILE_END: .tmp
	NUM_CLEAN_TESTS: 10
	AWK_FORMAT: awk/csvtojson.awk
	AWK_SHRINK: awk/consolidateIngreds.awk
	AWK_TRIM: awk/trim.awk
	AWK_RM_INGRDS: awk/removeIngreds.awk
	AWK_MAP: awk/map_fdcid_gtin.awk
	NOCATS_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/nocats.tmp
	PREPPED_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/prepped.tmp
	INGREDIENTS_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/ingredients.tmp
	INGREDIENTSB4_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/ingredientsb4.tmp
	NON_INGREDIENTS_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/non_ingredients.tmp
	SET_INGREDIENTS_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/set_ingredients.tmp
	RELEVANTDATA_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/relevantdata.tmp
	RELEVANTSORTED_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/relevantsorted.tmp
	RELEVANTTRIMMED_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/relevanttrimmed.tmp
	CLEANTEST_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/longestlines.tmp
	JOINED_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/joined.tmp
	SORTED_TMP: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/temp/sorted.tmp
	USAGE:
		Usage: ./convertToJson.sh [OPTION] (-h for help)
	HELP:
		Usage: ./convertToJson.sh [OPTION] (-h for help)
		OPTIONS:
			-b: bypass debug mode
			-s: print settings only
			-h: print help
		**debug mode: leave temp files undeleted**
		**debug mode is ON if no OPTION specified
