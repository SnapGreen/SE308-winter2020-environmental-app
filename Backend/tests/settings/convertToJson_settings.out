settings check:
	SETTINGS: /home/jtwedt/projSE308/SE308-winter2020-environmental-app/Backend/tests/settings.txt
	AWKDIR: 
	DATADIR: 
	FDADIR: 
	FDAINDIR: 
	FDAOUTDIR: 
	LOGDIR: 
	RAWDATADIR: 
	TMPDIR: 
	TESTDIR: 
	FDAINFILE: 
	MAPFILE: 
	PRODS_PER_JSON: 
	FDA_PATTERNFILE: 
	SPLIT_PREFIX: 
	SUFFIX_LEN: 
	OUTFILE_END: 
	TMPFILE_END: 
	NUM_CLEAN_TESTS: 
	AWK_FORMAT: csvtojson.awk
	AWK_SHRINK: consolidateIngreds.awk
	AWK_TRIM: trim.awk
	AWK_RM_INGRDS: removeIngreds.awk
	AWK_MAP: map_fdcid_gtin.awk
	NOCATS_TMP: nocats.tmp
	PREPPED_TMP: prepped.tmp
	INGREDIENTS_TMP: ingredients.tmp
	INGREDIENTSB4_TMP: ingredientsb4.tmp
	NON_INGREDIENTS_TMP: non_ingredients.tmp
	SET_INGREDIENTS_TMP: set_ingredients.tmp
	RELEVANTDATA_TMP: relevantdata.tmp
	RELEVANTSORTED_TMP: relevantsorted.tmp
	RELEVANTTRIMMED_TMP: relevanttrimmed.tmp
	CLEANTEST_TMP: longestlines.tmp
	JOINED_TMP: joined.tmp
	SORTED_TMP: sorted.tmp
	USAGE:
		Usage: ./convertToJson.sh [OPTION] (-h for help)
	HELP:
		Usage: ./convertToJson.sh [OPTION] (-h for help)
		OPTIONS:
			-b: bypass debug mode
			-s: print settings only
			-n: run with settings relative to repo root
			-h: print help
		**debug mode: leave temp files undeleted**
		**debug mode is ON if no OPTION specified
